package.json

```diff
@@ -100,8 +100,10 @@
     "react-dom": "17.0.2"
   },
   "dependencies": {
+    "@types/uuid": "8.3.4",
```

Comment:
Should we move this to `devDependencies`?

Reply:
Yes, we should